### DISCUSSION TITLE
Snapshot Reversion When Manager is Missing

### DISCUSSION THREAD

**Discussion by Daryl-Horton** [Comfy-Org/ComfyUI-Manager#461] [Q&A]
After updating ComfyUI, Manager went missing and loading a workflow displays all nodes as missing. Is it possible to revert using a Snapshot when Manager is missing or is the only way to do something similar to just reinstall ComfyUI and copy over the Snapshot to the new installation?

**Comment by ltdrdata** [comment]
If you copy the snapshot file you want to restore from the snapshots directory to the path `ComfyUI-Manager/startup-scripts/restore-snapshot.json`, it attempts to restore using that snapshot.

Since it's unclear at which stage your situation broke down, I can't be certain if this will be recoverable, but give it a try.

**Reply by Daryl-Horton** [reply]
that worked. Thank you!

**Reply by Daryl-Horton** [reply]
After reinstalling the updated nodes one by one, it looks like ComfyUI-Easy-Use is the node causing the Manager to go missing.
